app/Repositories/MicrosoftTeam/MicrosoftTeamRepository.php

```diff
@@ -170,8 +171,10 @@ public function createMSTeamAssignment($token, $classId, $project)
                     'Authorization' => 'Bearer ' . $token
                 ];
             
-                $response = Http::withHeaders($headers)->withOptions(["verify"=>false])->post($apiURL, $postInput);
+                $response = Http::withHeaders($headers)->withOptions(["verify"=>false])
```

Comment:
If this call fails 3 times then API will fail, need to find way to make sure to call after update is reflect on MS Team.